pkg/apis/apps/v1alpha1/servicebindingrequest_types.go

```diff
@@ -4,15 +4,16 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// NOTE: json tags are required.  Any new fields you add must have json tags for the fields to be serialized.
+// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
 
 // ServiceBindingRequestSpec defines the desired state of ServiceBindingRequest
 // +k8s:openapi-gen=true
 type ServiceBindingRequestSpec struct {
 	// Important: Run "operator-sdk generate k8s" to regenerate code after modifying this file
-	// Add custom validation using kubebuilder tags: https://book.kubebuilder.io/beyond_basics/generating_crd.html
+	// Add custom validation using kubebuilder tags:
+	// 	https://book.kubebuilder.io/beyond_basics/generating_crd.html
 
-	// BackingServiceSelector is used to identify the backing service operator.
+	// BackingSelector is used to identify the backing service operator.
```

Comment:
Is this to revert PR #45 ?

Reply:
Good spot, I misunderstood this. Let me fix it.

Reply:
@baijum please consider new changes, should be fixed now.